src/store/files.js

```diff
@@ -150,9 +150,11 @@ const actions = {
 	 *
 	 * @param {object} context the store mutations
 	 * @param {Array} files list of files
+	 * @return {Array} the appended files
 	 */
 	appendFiles(context, files = []) {
 		context.commit('updateFiles', files)
+		return files.filter(file => !!state.files[file.fileid])
```

Comment:
The proper way to do it would be

```suggestion
	 * @return {Array} the appended files
	 */
	appendFiles({ commit, state }, files = []) {
		commit('updateFiles', files)
		return files.filter(file => !!state.files[file.fileid])
```

Reply:
or `context.state.files`? .. had taken it from line 181, where `state` is also used directly.

Reply:
I see your proposal.. context has commit and files.. yes that makes sense

Reply:
> or `context.state.files`? .. had taken it from line 181, where `state` is also used directly.

Care to also change that? :see_no_evil: 
But yes, `context.state` also works :)